cmd/build.go

```diff
@@ -44,28 +45,27 @@ var buildCmd = &cobra.Command{
 		}
 
 		syscalls := make([]string, 0)
+		var syscallList = make(map[string]int)
 		for _, fileObj := range files {
-			//fmt.Println("[" + fileObj.Name() + "]")
-
 			file, err := os.Open(inputDirectory + "/" + fileObj.Name())
```

Comment:
filepath.Join is better than string concatenation.

But it's not related to this PR, and you may have other lines to fix

Reply:
Yea should be an easy thing to bulk-edit